charts/amazon-cloudwatch-observability/values.yaml

```diff
@@ -1185,8 +1185,14 @@ admissionWebhooks:
   ## Secret labels
   secretLabels: { }
 
+## List of AmazonCloudWatchAgent workloads to install & manage, each representing an independent installation of the AmazonCloudWatchAgent CustomResource. Each entry in this list uses the schema & the defaults from $.agent, so only provide any overrides here.
+agents:
+  - name: cloudwatch-agent
+
 agent:
   name:
+  mode: # represents the mode the cloudwatch-agent will run in (deployment, daemonset or statefulset)
+  replicas: # The total number non-terminated pods targeted by this cloudwatch-agent's deployment or statefulSet.
   image:
     repository: cloudwatch-agent
     tag: 1.300049.1b929
```

Comment:
need to update this

Reply:
Being done as part of https://github.com/aws-observability/helm-charts/pull/142